web/src/components/editor.tsx

```
@@ -259,8 +247,8 @@ function EditorContent() {
         },
     );
 
-    if (!isReady) {
-        return <WasmLoadingScreen progress={progress ?? 0} />;
+    if (true) {
```

Biome (2.4.6)

[error] Unexpected constant condition.

(lint/correctness/noConstantCondition)